.database_generator/generate_database.sh

```
@@ -62,20 +62,20 @@ cd /QuestieDB
 LAST_PATH="$(pwd)"
 
 # Goto the QuestieDB/.database_generator directory
-cd $FULL_PATH
+cd "$FULL_PATH"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)